src/webgpu/shader/execution/expression/call/builtin/texture_utils.ts

```diff
@@ -0,0 +1,747 @@
+/* eslint-disable prettier/prettier */
```

Comment:
There doesn't seem to be anything in this file that needs prettier disabled.

If there is, please use `/* prettier-ignore */` where needed instead of globally disabling prettier.
`// prettier-ignore` or `/* prettier-ignore */` applies to the entire AST subtree that immediately follows it.